src/js/call_list.js

```diff
@@ -1,7 +1,7 @@
 
 async function calculatePriorities()
 {
-    var callListsXML = loadXMLDoc(apiUrl+"/infected?compress=true");
+    var callListsXML = loadXMLDoc(apiUrl+"/infected?compress");
```

Comment:
As @Ordinateur-Hack mentioned, you have a double `/` here. The global variable contains one and here is one. Please remove that before merging this.

Reply:
Just fixed it. Ready for merging now